app/assets/stylesheets/application.css.scss

```diff
@@ -264,12 +266,12 @@ a.btn.btn-default:hover {
 #cta {
   color: #fff;
   background-color: #C9D6DF;
-                padding: 40px;
-
-                padding-bottom: 240px;
-                background: #d4d4d4 url('code_background.jpg');
-                                background-repeat: no-repeat;
-                margin-bottom: 50px;
+  padding: 40px;
+  background: #d4d4d4 url('code_background.jpg');
```

Comment:
Color literals like `#d4d4d4` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Prefer double-quoted strings